.env.appStore.example

```diff
@@ -125,5 +125,11 @@ SALESFORCE_CONSUMER_SECRET=""
 ZOHOCRM_CLIENT_ID=""
 ZOHOCRM_CLIENT_SECRET=""
 
+# ALBY
+# Used for the Alby payment app / receiving Alby payments
+# Get it from: https://getalby.com/developer/oauth_clients
+# Set callbackUrl to YOUR_APP_DOMAIN/api/integrations/alby/alby-webhook
+NEXT_PUBLIC_ALBY_CLIENT_ID=""
+NEXT_PUBLIC_ALBY_CLIENT_SECRET=""
```

Comment:
We don't need these env variables now. See, https://github.com/calcom/cal.com/blob/main/packages/app-store/CONTRIBUTING.md#others

Reply:
I will fix this tomorrow